### PR TITLE
[jsx/Markdown] Robust Upgrade of Markdown to Html

### PR DESCRIPTION
**Pull request by maltheism**
## Brief summary of changes

Package.json now includes [react-markdown](https://www.npmjs.com/package/react-markdown). The module has an MIT License and is actively maintained.

I modified webpack.config.js to remove the json rule. `Since webpack >= v2.0.0, importing of JSON files will work by default` [source](https://github.com/webpack-contrib/json-loader). Reason the json rule was removed in this PR was because react-markdown complained about it.

This PR will be helpful when merged and when https://github.com/aces/Loris/pull/5829 is merged as well. Since I'll be able to then modify the dashboard to use the react-markdown for converting the markdown string from admin configurations to html.

**Security Enhancement**: The react-markdown module allows "dangerouslySetInnerHTML" to be removed from the jsx/Markdown.js

Everything from markdown now will be parsed to the equivalent html.

#### Testing Instruction(s)
View the module help instructions and see how the Markdown renders.

#### Link(s) to related issue(s)

* Resolves https://github.com/aces/Loris/issues/5259

### CODE REVIEW

**Comment by johnsaigle** [comment]
This looks promising 👍Can you provide detailed testing instructions?

**Comment by maltheism** [comment]
Hi @johnsaigle, I added instructions.

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "prop-types": "^15.7.2",
     "react": "^16.11.0",
     "react-addons-create-fragment": "^15.6.2",
+    "react-markdown": "^4.2.2",
```
Comment (johnsaigle) [inline package.json]: Can you do `npm install` and commit the `package-lock.json` file?

Comment (maltheism) [reply package.json]: @johnsaigle I think master's package-lock.json isn't correct currently.

Comment (maltheism) [reply package.json]: I'm getting the git status message `modified: package-lock.json` after running npm install on aces/master and when trying to checkout to another branch.

Comment (maltheism) [reply package.json]: @johnsaigle I committed the package-lock.json since the new npm module updates it. I was a little confused at first. 🙂

Comment (johnsaigle) [reply package.json]: It looks like the new dependency isn't in that file though. It's just changing a few instances of http to https.

Comment (maltheism) [reply package.json]: @johnsaigle I'm clueless to why react-markdown wasn't in package-lock.json after running npm install in the previous commit. The latest commit now has the react-markdown inside the package-lock.json. I had to delete the `node_modules` & `package-lock.json` before running npm install.

**Pull request merged**